app/views/shared/_search_subnavbar.html.erb

```diff
@@ -38,7 +38,7 @@
 </div>
 <% if article_search? || current_page?(feedback_path) # For JS enabled and disabled %>
   <div id="connection-form" class="feedback-form-container collapse">
-    <%= render_feedback_form('connection') if show_feedback_form? %>
+    <%= render_connection_form if show_connection_form? %>
```

Comment:
Because this partial is never rendered for the FeedbackFormController, we can remove `show_connection_form?` entirely.

Reply:
Removed